data_steward/analytics/cdr_ops/README.md

```diff
@@ -4,4 +4,130 @@ Notebooks for operational evaluation of data.
 
 # Running
 
-Refer to `data_steward/notebooks/README.md`.
\ No newline at end of file
+Refer to `data_steward/notebooks/README.md`.
+
+# Notebook Execution
+
+Jupytext notebooks (.py extension) can be executed with supplied parameters and exported to an HTML document using the utility `report_runner.py`.
+
+```
+$ python report_runner.py -h
+usage: report_runner.py [-h] [--output_path OUTPUT_PATH] [--help_notebook]
+                        [--params PARAM_NAME PARAM_VALUE]
+                        notebook_path
+
+positional arguments:
+  notebook_path         A .py jupytext file
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --output_path OUTPUT_PATH
+                        An output .html file. If not provided, defaults to an
+                        html path with name of [notebook path]
+  --help_notebook       Lists the accepted parameters for [notebook_path]
+  --params PARAM_NAME PARAM_VALUE, -p PARAM_NAME PARAM_VALUE
+                        A parameter to pass to [notebook path] (multiple may
+                        be provided)
+```  
+
+### Preparing Notebook Template
+
+In order for an input notebook to accept parameters, it must be prepared in the special format described in the [papermill](https://papermill.readthedocs.io/en/latest/usage-parameterize.html) package documentation.  
+
+Within an opened jupyter notebook, do the following:
+1. Select the `View -> Cell Toolbar -> Tags` menu option to activate the notebook's tag view.
+2. In a single cell, define 1 or more variables that will be your parameters.  
+    1. A optional comment on a variable's line will serve as its description.
+    1. An optional type hint can be provided. 
+3. In your parameter-containing cell, enter the term **parameters** into the textbox adjacent to the <em>Add tag</em> button.
+4. Click the <em>Add tag</em> button to mark the cell with the entered tag.
+
+
+![Parameterization instructions](./images/parameter_definition.png)
+
+### Viewing Notebook Parameters
+
+To view the parameters required for a particular notebook, the <em>--help_notebook</em> option can be passed to `report_runner.py`.
+
+```
+$ python report_runner.py "Example Notebook.py" --help_notebook
+Parameters inferred for notebook Example Notebook:
+2021-01-28 17:44:08 - INFO - papermill - Input Notebook:  Example Notebook.ipynb
+  my_unknown_parameter: Unknown type (default "", required=True)
+                                  This parameter has an unknown type.
+  my_string_parameter: str (default "", required=True)
+                                  This is a string parameter.
+  my_int_parameter: int (default 0, required=False)
+                                  This is an integer parameter.
+```
+
```

Comment:
Verified this was working by executing the following command:

`python report_runner.py "rdr_export_qc.py" --help_notebook`